flair/trainers/sequence_tagger_trainer.py

```diff
@@ -175,38 +171,46 @@ def evaluate(self, evaluation: List[Sentence], out_path=None, evaluation_method:
                 # Step 3. Run our forward pass.
                 score, tag_seq = self.model.predict_scores(sentence)
 
+                # add predicted tags
+                for (token, pred_id) in zip(sentence.tokens, tag_seq):
+                    token: Token = token
+                    # get the predicted tag
+                    predicted_tag = self.model.tag_dictionary.get_item_for_index(pred_id)
+                    token.add_tag('predicted', predicted_tag)
+
+                gold_tags = []
+                predicted_tags = []
+
+                # get spans
+                for tag in sentence.get_spans(self.model.tag_type):
+                    gold_tags.append(tag.__str__())
```

Comment:
Instead of `tag.__str__()` you should use `str(tag)`.

Reply:
Could actually be rewritten to:

`gold_tags = [str(tag) for tag in sentence.get_spans(self.model.tag_type)]`